bundles/com.espressif.idf.ui/META-INF/MANIFEST.MF

```diff
@@ -29,10 +29,19 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.launchbar.ui,
  org.eclipse.ui.console,
  org.eclipse.jgit,
- org.eclipse.swtchart,
- org.eclipse.cdt.cmake.is.core.ui
+ org.eclipse.swtchart
 Automatic-Module-Name: com.espressif.idf.ui
 Bundle-ActivationPolicy: lazy
 Bundle-RequiredExecutionEnvironment: JavaSE-11
 Export-Package: com.espressif.idf.ui,
- com.espressif.idf.ui.dialogs
+ com.espressif.idf.ui.dialogs,
+ com.espressif.idf.ui.handlers,
+ com.espressif.idf.ui.help,
+ com.espressif.idf.ui.install,
+ com.espressif.idf.ui.preferences,
+ com.espressif.idf.ui.size,
+ com.espressif.idf.ui.templates,
+ com.espressif.idf.ui.update,
+ com.espressif.idf.ui.wizard
```

Comment:
I think we can remove them if it's not used anywhere else

Reply:
I removed them. It was added automatically and I forgot to remove it. Sorry for that